Editor/Completion/XmlCompletionCommitManager.cs

```diff
@@ -38,10 +38,39 @@ public CommitResult TryCommit (IAsyncCompletionSession session, ITextBuffer buff
 			}
 
 			switch (kind) {
+			case XmlCompletionItemKind.SelfClosingElement: {
+					if (typedChar != '>') {
+						string insertionText = $"{item.InsertText}/>";
+						Insert (session, buffer, insertionText);
+						ShiftCaret (session, 2, XmlCaretDirection.Left);
+						return CommitResult.Handled;
+					} else {
+						return CommitResult.Unhandled;
```

Comment:
When the typed char is `>` it should insert the closing tag i.e. `goto XmlCompletionItemKind.Element`

Reply:
Made the change